src/libraries/Common/src/Interop/Interop.TimeZoneInfo.Browser.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Would be better to call the implementation `Interop.TimeZoneInfo.Invariant.cs` ?

Reply:
Actually looking at the code a little bit more I think we should fix the caller instead. Let's make Browser (or Invariant) version of `GetDisplayName`

Reply:
I agree, that was what I was trying to suggest in one of my earlier comments :)

Reply:
@marek-safar we still will have to introduce `Interop.TimeZoneInfo.Browser.cs` to hide `GetTimeZoneDisplayName` pinvoke - we don't need a `[DllImport]` that points nowhere - the pinvokegenerator will assert.

Reply:
we can exclude `Interop.TimeZoneInfo.cs` for Browser but then I'll have to ifdef out the call for it or extract more code to a separate file - is it worth it?

Reply:
Yes and it won't be for browser only in the future. The calling code is oddly complex to let's replace it with single line

Reply:
addressed